docs/vsphere_ci/platform-none/README.md

```diff
@@ -0,0 +1,36 @@
+# Adding PTR records for platform=none CI clusters in vSphere
+
+WMCO' CSRs approver requires reverse DNS lookup for each Windows instance that 
+joins the cluster as a Windows worker. For CI clusters with platform-agnostic
+infrastructure (platform=none) a fixed lease pool is configured to ensure latter 
+requirement is met.
+
+The [create-ptr-records.sh](create-ptr-records.sh) script creates the reverse
+DNS lookup by creating PTR records for each IP address available in the selected
+subnets.
+
+vSphere CI (`vmc-ci.devcluster.openshift.com`) uses AWS Route53 for DNS
+resolution and the following ci-segments are reserved for platform=none clusters
+- ci-segment-56
+- ci-segment-57
+- ci-segment-58
+- ci-segment-59 
+
+each segment sits on a `/27` subnet (`192.168.x.1/27`) with DHCP range from
+`192.168.x.10` to `192.168.x.30`, where `x` accounts for the third octet and 
+matches the number of the ci-segment, from 56 to 59 inclusive. DNS is provided
+by the VPC with server IP `10.0.0.2`.
```

Comment:
This information might fit better as comments within the script. Its a bit odd to have it in public docs.

Reply:
[vsphere_ci](https://github.com/openshift/windows-machine-config-operator/tree/90c6705ee718c644a59743b90eb2460ede1d4fed/docs/vsphere_ci) is for WinC internal use online, is not public docs. Meaning is not pushed to the  [Openshift docs](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.html#preparing-vsphere-for-windows-containers).

Reply:
But this does not preclude them from being better suited to having them as comments in the script.

Reply:
Moved this section to the script.